src/runtime/app.ts

```diff
@@ -114,15 +114,15 @@ export function create(): App {
     settings: settingsComponent.public,
     schema: schemaComponent.public,
     server: serverComponent.public,
-    assemble() {
+    async assemble() {
```

Comment:
Its async now... 👋

Reply:
It's async because of a single `existsAsync`. Are we sure it's worth the potential trouble? Keeping the assembly sync seems pretty crucial?

Reply:
If you want to keep it async, then you probably need to update the start-module as well to run `app.start` only after `app.assemble` is done running?

```ts
app.assemble().then(app.start)
```